addons/static/templates/policy_cache_transformer_persona.json

```diff
@@ -874,5 +874,226 @@
       ],
       "actions": ["select"]
     }
+  ],
+  
+  "persona-aiasset-read": [
+    {
+      "policyType": "ACCESS",
+      "policyResourceCategory": "ENTITY",
+      "resources": [
+        "entity:{entity}",
+        "entity:{entity}/*",
+        "entity-type:AIApplication",
+        "entity-type:AIModel",
+        "entity-classification:*"
+      ],
+      "actions": ["entity-read"]
+    }
+  ],
+  "persona-aiasset-create": [
+    {
+      "policyResourceCategory": "ENTITY",
+      "policyType": "ACCESS",
+      "resources": [
+        "entity:{entity}",
+        "entity:{entity}/*",
+        "entity-type:AIApplication",
+        "entity-type:AIModel",
+        "entity-classification:*"
+      ],
+      "actions": ["entity-create"]
+    }
+  ],
+  "persona-aiasset-update": [
+    {
+      "policyResourceCategory": "ENTITY",
+      "policyType": "ACCESS",
+      "resources": [
+        "entity:{entity}",
+        "entity:{entity}/*",
+        "entity-type:AIApplication",
+        "entity-type:AIModel",
+        "entity-classification:*"
+      ],
+      "actions": ["entity-update"]
+    },
+    {
+      "policyResourceCategory": "RELATIONSHIP",
+      "policyType": "ACCESS",
+      "description": "Link/unlink AIModel to any AIApplication",
+      "resources": [
+        "relationship-type:*",
+        "end-one-entity-type:AIApplication",
+        "end-one-entity-classification:*",
+        "end-one-entity:*",
+
+        "end-two-entity-type:AIModel",
+        "end-two-entity-classification:*",
+        "end-two-entity:{entity}",
+        "end-two-entity:{entity}/*"
```

Comment:
Referring to ai_applications_ai_models at https://github.com/atlanhq/models/pull/1385/files

`end-one-entity-type` should be `AIModel` & `end-two-entity-type` should be `AIApplication`